tools/mksurfdata_map/src/mksurfdat.F90

```diff
@@ -1127,6 +1130,9 @@ program mksurfdat
        ! Read in each dynamic pft landuse dataset
 
        nfdyn = getavu(); call opnfil (mksrf_fdynuse, nfdyn, 'f')
+       
+       ! IV read in dynamic lake dataset
+       nfdynlak = getavu(); call opnfil (mksrf_fdynlak, nfdynlak, 'f')
```

Comment:
What happens if the dynamic lake file isn't provided? We need a way for it to abort. We also should decide on if lakes should always be dynamic for transient conditions, or if we you pick and choose? It would probably be best to always have it on the datasets, but then decide in CTSM whether to activate it or not. We have something similar for harvest.

Reply:
As I mention in a comment below, I'm thinking we should put the lake file names on the same file as the pft & harvest file names. @ekluzek does that make sense to you, or do you see a reason to keep them on separate files? My assumption (like yours, @ekluzek ) is that we'll always put the transient lake data on the landuse timeseries files, but this will only be activated if the flag `do_transient_lakes` is true. By default I think this will be false for now, while this is still an experimental setting.